include/CL/opencl.hpp

```diff
@@ -4534,6 +4543,24 @@ class BufferRenderGL : public Buffer
     }
 };
 
+#if defined(cl_ext_image_requirements_info)
+enum class ImgReqInfoExt : cl_image_requirements_info_ext
+{
+    ImgReqNone = 0,
+    ImgReqRowPitchAlign = CL_IMAGE_REQUIREMENTS_ROW_PITCH_ALIGNMENT_EXT,
```

Comment:
I think we should rethink this naming convention slightly since the current full enum name will be `ImgReqInfoExt::ImgReqNone`, which is a little redundant.

Could this just be:

`ImgReqInfo::NoneExt`?

* Don't need to duplicate `ImgReq` on the enum name itself when it is already in the enum class name.
* Move `Ext` to the enum name rather than the enum class name?
* We might also want to consider spelling out `Image` or `Requirements` (or both)?